pkg/kubectl/cmd/config/config.go

```diff
@@ -66,8 +63,7 @@ func NewCmdConfig(pathOptions *PathOptions, out io.Writer) *cobra.Command {
 
 	// file paths are common to all sub commands
 	cmd.PersistentFlags().BoolVar(&pathOptions.Local, "local", pathOptions.Local, "use the kubeconfig in the current directory")
-	cmd.PersistentFlags().BoolVar(&pathOptions.Global, "global", pathOptions.Global, "use the kubeconfig from "+pathOptions.GlobalFile)
-	cmd.PersistentFlags().BoolVar(&pathOptions.UseEnvVar, "envvar", pathOptions.UseEnvVar, "use the kubeconfig from $"+pathOptions.EnvVar)
+	cmd.PersistentFlags().BoolVar(&pathOptions.Global, "global", pathOptions.Global, "use the kubeconfig from ~"+clientcmd.RecommendedHomeFileName)
```

Comment:
The help string for `--global` should mention that if the envvar is set, it will be used instead of the default global file.